src/components/BasicNodeViewer/BasicNodeViewer.tsx

```diff
@@ -21,12 +21,12 @@ interface BasicNodeViewerProps {
 }
 
 export const BasicNodeViewer = ({node, additionalNodesProps, className}: BasicNodeViewerProps) => {
-    let nodeHref: string | undefined;
+    let nodeRef: string | undefined;
 
     if (additionalNodesProps?.getNodeRef) {
-        nodeHref = additionalNodesProps.getNodeRef(node) + 'internal';
+        nodeRef = additionalNodesProps.getNodeRef(node) + 'internal';
     } else if (node.NodeId) {
-        nodeHref = createDeveloperUILinkWithNodeId(node.NodeId) + 'internal';
+        nodeRef = createDeveloperUILinkWithNodeId(node.NodeId) + 'internal';
     }
```

Comment:
It's actually `nodeHref`. It looks like a typo in initial function, ref in React is a completely different thing